ndbench-es-plugins/src/test/java/com/netflix/ndbench/plugin/es/EsRestPluginIntegrationTest.java

```diff
@@ -0,0 +1,152 @@
+package com.netflix.ndbench.plugin.es;
```

Comment:
Do we want to have a test subpackage?

Reply:
Tests are under the `test` folder anyway, and sharing the package with the code they test allows package private access which can sometimes be useful.

Reply:
Tests are under the `test` folder anyway, and sharing the package with the code they test allows package private access which can sometimes be useful.

Reply:
yes. as Gennadiy said..  i have subpackages so that stuff that  is in package scope will be visible and can be tested.. but is not part of the 'public interface' of the class in question.